### DISCUSSION TITLE
Warn when a plugin takes too long to build for live reload

### DISCUSSION THREAD

**Discussion by Emmanuel-Messulam** [mkdocs/mkdocs#3641] [Ideas]
### Description

Live reload should be as fast as possible. It would be a good idea to time the different plugins at different stages of page building, so that a warning is issued if a plugin takes too long.

### Use Cases

Mostly with large sites, and with a few heavy plugins. It would be useful which one is the slowest to replace it.

### Screenshots / Mockups

```
WARNING  -  It seems that plugin aaa-bbb-plugin took 2.5 seconds to build a.md, this might slow down live reloads.
```

**Comment by lovelydinosaur** [comment]
*Aside*: I've converted this to a discussion rather than an issue... we could do with some better process here, so that we're helping the community start with design discussions, and can then escalte into issues if we've decided there *is* something we want to action.
